lib/common/formats.js

```diff
@@ -52,6 +52,24 @@ function iconsWithPrefix(prefix, properties, config) {
     })
     .value().join('\n');
 }
+
+function filterValueProperty(properties, path, obj ) {
+  path = path || [];
```

Comment:
I believe there is a way to do this without passing keeping a path around...

I think @chazzmoney wrote this in our demo repo, let me know if this makes sense:
```js
const minifyDictionary = function(obj) {
  let toRet = {};
  if (_.has(obj, 'value')) {
    return obj.value;
  } else {
    for(var name in obj) {
      toRet[name] = minifyDictionary(obj[name]);
    }
  }
  return toRet;
};
```

Reply:
oh nice, let me take a look into this function

Reply:
I wrote that? Seems like something I'd do. You might need a if(obj.hasOwnProperty(name)) wrapper on that "toRet[name] = " assignment statement.